life_expectancy/tests/test_cleaning.py

```diff
@@ -1,13 +1,13 @@
 """Tests for the cleaning module"""
 import pandas as pd
 
-from life_expectancy.cleaning import clean_data
+from life_expectancy.cleaning import main
 from . import OUTPUT_DIR
 
 
```

Comment:
You shall provide a test for the region filter, that is another requirement included by the assignement
Here is an example:
```suggestion
def test_location_filter(get_configs):
    """Run the `clean_data` function and compare the output to the expected output"""

    clean_data('ES')

    location_col = get_configs.get('col_location', 'region')

    # did not change the path, but we should
    es_filter = np.unique(pd.read_csv(
        OUTPUT_DIR / "pt_life_expectancy.csv"
    )[location_col].to_numpy())

    nok_msg = "clean_data location filtering is not working"
    assert len(es_filter) == 1, nok_msg
    assert es_filter[0] == 'ES', nok_msg
```